### PR TITLE
Adapt Spine, Erizo Agent and Basic Example to jshint

### PR DESCRIPTION
**Pull request by jcague**

### CODE REVIEW

File: extras/basic_example/public/script.js [hunk 1]
```diff
@@ -55,66 +63,65 @@ window.onload = function () {
...
         for (var index in streams) {
           var stream = streams[index];
           if (localStream.getID() !== stream.getID()) {
             room.subscribe(stream);
-            stream.addEventListener("bandwidth-alert", function (evt){
-                console.log("Bandwidth Alert", evt.msg, evt.bandwidth);
-            });
-            
-
+            stream.addEventListener('bandwidth-alert', cb);
```
Comment (lodoyun) [inline extras/basic_example/public/script.js]: Just curious, what's wrong with having the callback function here?

File: spine/erizofc.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
+/*
```
Comment (lodoyun) [inline spine/erizofc.js]: This file is copied in installSpine.sh, should be in .gitignore

Comment (jcague) [inline spine/erizofc.js]: good catch! I'll check it, I added all the folder to save time 😞

File: extras/basic_example/public/script.js (hunk 1 above)
Comment (jcague) [inline extras/basic_example/public/script.js]: Creating functions within a loop is discouraged in general: http://airbnb.io/javascript/#functions--in-blocks

**Pull request merged**